packages/scripts/run-heavyweight-dedupe-test.sh

```
@@ -0,0 +1,119 @@
+git checkout -
+echo ">>> 4. Generating deduplicated files from feature-branch \`$(git rev-parse --abbrev-ref HEAD)\`"
+
+printf "\r\033[K"
+printf "Running \`npm run build\`..."
+npm run build 1>/dev/null 2>&1
+
+printf "\r\033[K"
+printf "Running \`SOME SCRIPT\`..."
+
+# Triggers cleanup
+popd
```

Shellcheck (0.10.0)

[warning] Use 'popd ... || exit' or 'popd ... || return' in case popd fails.

(SC2164)